### ISSUE TITLE
I can't install the environment cause many packages not found.

### ISSUE THREAD

**Issue by liyunjian** [jbloomlab/SARS-CoV-2-RBD_MAP_Crowe_antibodies#15]
this is one error example:
ResolvePackageNotFound: 
  - libgfortran-ng==7.5.0=hdf63c60_6

**Comment by jbloom** [comment]
Yes, retrospectively the idea of doing a completely pinned environment as in the `environment.yml` file is not a great idea as packages become outdated or don't work across machines.

Does it work if you instead build the environment in `environment_unpinned.yml`?

**Comment by liyunjian** [comment]
After two days of trying, I deleted the wrong entries in environment.yml, such as libgfortran-ng==7.5.0=hdf63c60_6, and then imported the environment successfully, and then I manually installed some required versions in turn in subsequent runs package. For the time being, it runs without error.

**Comment by liyunjian** [comment]
But I encountered a new problem, the quality score in the FASTQ sequence is 30, and it is displayed as "?" in the downloaded FASTQ file. But an error was reported when executing the code. It seems that dms_variants.illuminabarcodeparser considers this an error? （dms-variants==0.8.4）Please can you help me solve it.

**Comment by jbloom** [comment]
If you can provide more specific details on the error message I will take a look.
